python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -131,6 +131,27 @@ def get_node_unique_id(self, *, parent_unique_id: Optional[str], index: Optional
         parts = [str(parent_unique_id), str(index), self.name]
         return non_secure_md5_hash_str("".join(parts).encode())
 
+    def get_alternate_node_unique_ids(
```

Comment:
consider replacing 'alternate' with 'backcompat' to make it clearer why one would have these set